src/vscodeAdapter.ts

```diff
@@ -299,6 +299,100 @@ export interface TextEditor {
      */
     document: any;
 }
+/**
+	 * A universal resource identifier representing either a file on disk
+	 * or another resource, like untitled resources.
+	 */
+export interface Uri {
```

Comment:
I'm assuming this is effectively a copy from vscode.d.ts? How are we going to keep it in sync? If the answer is "the build breaks if we take new dependencies from an update vscode.d.ts", that's acceptable to me.